### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by ayesha54**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,6 +1,9 @@
 class ApplicationController < ActionController::Base
   before_action :configure_permitted_parameters, if: :devise_controller?
-
+  protect_from_forgery with: :exception
```
Comment (ali-ehmed) [inline app/controllers/application_controller.rb:3]: spaces bw lines

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+  def hello
```
Comment (ali-ehmed) [inline app/controllers/application_controller.rb]: Why this method?

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <div id="navbar">
+      <ul class="nav navbar-nav">
+        <li><%= link_to 'Home', root_path %></li>
+      </ul>
+      <ul class="nav navbar-nav pull-right">
+        <% if user_signed_in? %>
+          <li class="dropdown">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+              <%= current_user.email %>
+
```
Comment (ali-ehmed) [inline app/views/layouts/_header.html.erb:15]: No space required.

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -10,6 +10,9 @@
   </head>
 
   <body>
+    <div class="container">
```
Comment (ali-ehmed) [inline app/views/layouts/application.html.erb:12]: indentation errors

**Comment by ali-ehmed** [comment]
Looks good.

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (ayesha54) [reply app/controllers/application_controller.rb:3]: umm... i write dirty code :/ but i will improve.

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (ayesha54) [reply app/controllers/application_controller.rb]: I was trying to render the page... ...

File: app/views/layouts/_header.html.erb (hunk 3 above)
Comment (ayesha54) [reply app/views/layouts/_header.html.erb:15]: I'll resolve this

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (ayesha54) [reply app/views/layouts/application.html.erb:12]: i'll resolve

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (ali-ehmed) [reply app/controllers/application_controller.rb]: Why? There ain’t need for it. Since root is fruits#index. Remove it

**Pull request merged**